### PR TITLE
SP-2 Created model

### PR DESCRIPTION
**Pull request by E-zh**
Added model, views and, controller and methods for viewing in Adminpanel. We can publish the views.

### CODE REVIEW

File: src/Models/StaticPages.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Ourgarage\StaticPages\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class StaticPages extends Model
+{
+    protected $table = 'static-pages';
+
+    public $timestamps = true;
```
Comment (garf) [inline src/Models/StaticPages.php]: Not necessary. It is true by parent class already.

File: src/Models/StaticPages.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Ourgarage\StaticPages\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class StaticPages extends Model
+{
+    protected $table = 'static-pages';
```
Comment (garf) [inline src/Models/StaticPages.php]: Do not mane tables with `-` symbol. Only underscore '_'

File: src/database/migrations/2016_04_22_045536_create_staticpages_table.php [hunk 3]
```diff
@@ -15,7 +15,7 @@ public function up()
         Schema::create('static-pages', function (Blueprint $table) {
```
Comment (garf) [inline src/database/migrations/2016_04_22_045536_create_staticpages_table.php]: `static_pages`

Comment (garf) [inline src/database/migrations/2016_04_22_045536_create_staticpages_table.php]: Recreate this migration please. File will be named differently,

File: src/resources/views/page_list.blade.php [hunk 4]
```diff
@@ -0,0 +1,17 @@
+@extends('admin.main')
```
Comment (garf) [inline src/resources/views/page_list.blade.php]: for view files let's use `pages-list.blade.php`

File: src/Models/StaticPages.php (hunk 1 above)
Comment (E-zh) [inline src/Models/StaticPages.php]: Done.

File: src/Models/StaticPages.php (hunk 2 above)
Comment (E-zh) [inline src/Models/StaticPages.php]: Ok

File: src/database/migrations/2016_04_22_045536_create_staticpages_table.php (hunk 3 above)
Comment (E-zh) [inline src/database/migrations/2016_04_22_045536_create_staticpages_table.php]: Done.

**Pull request merged**